python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -967,10 +970,16 @@ def _setFlushed(self):
         """Un-Set the dirty flag all of the way down the schema."""
         if self._schema:
             for k in self._schema.allItemNames():
+                ## Avoid attributes the likes of job.master which crawl back up the tree
+                if self._schema[k].hasProperty('visitable'):
+                    if (not self._schema[k].getProperties()['visitable'] or self._schema[k].getProperties()['visitable'] == 0):
+                        continue
+                if self._schema[k].hasProperty('transient'):
+                    if(self._schema[k].getProperties()['transient'] or self._schema[k].getProperties()['transient'] == 1):
```

Comment:
Yes and it was changed. I was changed I was just irked that when I type ```is True``` it doesn't work as expected due to historical reasons which are no longer valid.